### PR TITLE
Add quarantine media API test

### PR DESCRIPTION
**Pull request by erikjohnston**
Tests: https://github.com/matrix-org/synapse/pull/2292

### CODE REVIEW

File: tests/51media/48admin-quarantine.pl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+         $content_id = "$server$path";
+
+         matrix_send_room_message( $user, $room_id,
+            content => { msgtype => "m.image", body => "test.png", url => "$content_uri" }
+         );
+      })->then( sub {
+         do_request_json_for( $admin,
+            method  => "POST",
+            uri     => "/r0/admin/quarantine_media/$room_id",
+            content => {}
```
Comment (leonerd) [inline tests/51media/48admin-quarantine.pl]: Missing trailing `,`; also more below

File: tests/51media/48admin-quarantine.pl [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+         )
```
Comment (leonerd) [inline tests/51media/48admin-quarantine.pl]: Missing `;`; also more below

**Comment by richvdh** [comment]
holy necrophilia, batman

**Comment by ara4n** [comment]
itym necromancy :P

**Pull request merged**